Demo/HYPForms/HYPSampleCollectionViewController.m

```diff
@@ -233,15 +230,7 @@ - (void)readOnly:(UISwitch *)sender
 {
     [self.dataSource disable:sender.isOn];
 
-    HYPFormTarget *target = [HYPFormTarget new];
-    target.targetID = @"image";
-    target.typeString = @"field";
-
-    if (sender.isOn) {
-        target.actionTypeString = @"hide";
-    } else {
-        target.actionTypeString = @"show";
-    }
+    HYPFormTarget *target = [HYPFormTarget hideFieldTargetWithID:@"image"];
```

Comment:
Should be fixed by https://github.com/hyperoslo/HYPForms/commit/2a318b83ba05cbf42cab092fc0f3d0f9a37d6255